packages/Guard/src/GuardConnector.js

```diff
@@ -26,14 +36,6 @@ function GuardConnector({ group, isDirty }) {
   return null;
 }
 
-GuardConnector.propTypes = {
-  isDirty: PropTypes.bool,
-  group: PropTypes.string,
-};
-
-GuardConnector.defaultProps = {
-  isDirty: false,
-  group: null,
-};
-
-export default GuardConnector;
+GuardConnector.displayName = "GuardConnector";
```

Comment:
For all of the `displayNames`, the "consumer facing" name is what we've been using.  So, `Guard.Connector`.